djangomain/views.py

```diff
@@ -1,11 +1,27 @@
+from django.shortcuts import render
 from django.views.generic.base import TemplateView
 from drf_yasg import openapi
 from drf_yasg.views import get_schema_view
+from guardian.shortcuts import get_objects_for_user
 from rest_framework import permissions
 
+from station.models import Station
+
 
 class HomePageView(TemplateView):
-    template_name = "home.html"
+    """
+    View for displaying the home page and map dash app.
+    """
+
+    def get(self, request, *args, **kwargs):
+        from .dash_apps.finished_apps import stations_map
+
+        stations = get_objects_for_user(
+            request.user, "view_measurements", klass=Station
```

Comment:
I'm not entirely sure but I think the correct permission here is `view_stations`. Otherwise I need to be logged in to see anything (at least with the permissions currently assigned to the stations). 

@tsmbland , could you confirm what the right permissions to check are in order to view stations metadata?

Reply:
Ah yes good point. This should be `view_station`